### DISCUSSION TITLE
What do you like to see in the next milestone?

### DISCUSSION THREAD

**Discussion by sarahelsaig** [OrchardCMS/OrchardCore.Commerce#443] [Q&A]
Hi everyone,

We'd like to ask for your input in planning for the development up to OrchardCore.Comerce v3.0. What would you like to see added this year? What are the pain points that should be addressed (please provide scenarios or suggestions)? If you want us to prioritize an existing feature request, now is also the time to make some noise about it.

EDIT: ping @agriffard @DotCat1985 @dpomt  @effapp @giantpenguin @havana7 @hishamco @infofromca

**Comment by DotCat1985** [comment]
Hi,
I'd like support for [Satispay](https://www.satispay.com/en-it/) payments.

**Reply by Piedone** [reply]
Can you tell what's the benefit of Satispay over Stripe and Exactly, which OCC currently supports? Have you tried Exactly, which is also an EU payment provider and cheaper for EU companies than Stripe?

**Reply by DotCat1985** [reply]
Satispay provides:
-zero-transaction fees for buyers
-lower trasactions fees for vendors than another competitors, like PayPal
-excellent customer care
-payment by passing the QR code
You have to know that Stripe doesn’t support Satispay.

**Reply by Piedone** [reply]
I think Exactly does these too.

**Comment by dpomt** [comment]
Hi,

great to hear to OCC gets attention again. Thanks so much,

For me, I have four topics:

1) https://github.com/OrchardCMS/OrchardCore.Commerce/issues/397#issuecomment-1896291805

2) https://github.com/OrchardCMS/OrchardCore.Commerce/issues/4#issuecomment-1896294202

3) https://github.com/OrchardCMS/OrchardCore.Commerce/discussions/401#discussioncomment-8159899

4) Feature Request: Coupons
Coupons can expire, be private (to a single user or group). Customizable if applied to gross or net price.
Either percentage coupon or fixed discount (gross or net)

Thanks,
Dieter

**Comment by daerogami** [comment]
Coming from someone who only spun up an instance to evaluate, it seems like it's missing a few important features in the way of management on the admin UI.

- Dashboard for simple reports like orders by month, preview of recent orders, and some general stats such as total unfulfilled orders, active users, count of products with low inventory.
- Dedicated sections for orders and another for products. It might just be me, but using the content section to manage those seems counter-intuitive. I understand orders and products are built from the same base objects as other content, but I imagine our average client being confused by this.

More than happy to help contribute if these make sense.

**Comment by DotCat1985** [comment]
I'd like the localization of the routes like this, in order to manage multi-language e-commerce.

/en/cart
/it/carrello

/en/checkout
/it/checkout

/en/checkout/empty
/it/checkout/carrello-vuoto

**Comment by effapp** [comment]
Hi,
I'd like the support for multi user addresses .
The user can select address instead of entering new address when shopping cart check out.
